src/test/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.jpa.hibernate.ddl-auto=create-drop
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL95Dialect
+spring.jpa.properties.hibernate.format_sql=true
+# removes strange exception on application context load time (although it does not cause application crash)
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
```

Comment:
enable in IDEA:

- [x] Ensure line feed at file end on Save

and add newline